src/ripple/peerfinder/tests/PeerFinder_test.cpp

```diff
@@ -119,9 +122,18 @@ class Logic_test : public beast::unit_test::suite
             c.listeningPort = 1024;
             logic.config(c);
         }
+
+        std::array<std::uint8_t, 33> const key =
+        {
+            0x02, 0x70, 0xEB, 0x12, 0x56, 0x98, 0x14, 0xEB,
+            0xDD, 0x80, 0x91, 0x49, 0xFD, 0x8B, 0xF8, 0xF2,
+            0x45, 0x61, 0xF5, 0x32, 0x13, 0x86, 0xDE, 0xE7,
+            0xFF, 0xC6, 0xFD, 0xAC, 0xB7, 0x1C, 0x89, 0x0E,
+            0xE3
+        };
```

Comment:
Clang 3.7 issues the warning: `PeerFinder_test.cpp:128:13: warning: suggest braces around initialization of subobject [-Wmissing-braces]` We can use `{{` to silence the warning, although I find that code worse than what's here. Three options:

1) Disable the `missing-braces` warning rather than make the code worse
2) Live with the warning in clang 3.7 (other compilers don't seem to warn). There is an open issue for this in clang, it may get resolved in future versions: https://llvm.org/bugs/show_bug.cgi?id=21629
3) Initialize with `{{`